.github/workflows/cpu-inference.yml

```diff
@@ -0,0 +1,68 @@
+name: cpu-inference
```

Comment:
CPU workflow, still WIP

Reply:
@delock, any update on the workflow? Is this PR ready for review?

Reply:
@tjruwase Yes, this PR is ready for review.   We have a minimum set for CPU workflow that can pass and will need your comments on what should be included in CPU workflow.